game.lua

```diff
@@ -1,15 +1,20 @@
 --- Game related variables and functions.
 ...
 
+--- The current level the game is played on. Read-only.
+level = nil
+
+--- The amount of players in the game. Read-only.
+playersCount = 1
+
 --- The event handler for game start.
+--- @usage game.start.addListener(function() player.removeitem() player.chat("bye bye halo :)") end)
```

Comment:
These examples doesn't seem to provide much value.

```suggestion
--- @see eventlistener
```